apps/prairielearn/src/tests/accessibility/index.test.ts

```diff
@@ -151,12 +155,14 @@ const SKIP_ROUTES = [
   '/pl/course_instance/:course_instance_id/news_item/:news_item_id/*',
   '/pl/course/:course_id/cacheableElementExtensions/:cachebuster/*',
   '/pl/course/:course_id/cacheableElements/:cachebuster/*',
+  '/pl/course/:course_id/sharedElements/course/:producing_course_id/cacheableElements/:cachebuster/*',
   '/pl/course/:course_id/clientFilesCourse/*',
   '/pl/course/:course_id/course_admin/file_download/*',
   '/pl/course/:course_id/news_item/:news_item_id/*',
   '/pl/course/:course_id/question/:question_id/clientFilesCourse/*',
   '/pl/course/:course_id/question/:question_id/clientFilesQuestion/*',
   '/pl/course/:course_id/elements/*',
+  '/pl/course/:course_id/sharedElements/course/:producing_course_id/elements/*',
```

Comment:
I was keeping this list alphabetized; let's do the same with these changes.

Reply:
It doesn't seem particularly alphabetized--even in the snippet here, you can see the `/elements` URL appears after the `/question` one, which is not in order. the `grading_job` one also seems out of order. 

I mostly noticed this because I tried to solve the problem just by running the list through the command line `sort` tool, but that made way more changes than intended Maybe we should have a separate PR to just do that to the whole list? 

Anway, I changed the newly added lines to be as in-order as I could given the situation.